### DISCUSSION TITLE
Boundary condition error

### DISCUSSION THREAD

**Discussion by Dewu-Yang** [loganoz/horses3d#161] [Q&A]
Dear all,

    I am a beginner in HORSES3D, and I am currently facing an issue. When I import a new mesh and define the boundary conditions in the .control file, I encounter the following error. The boundary names defined in my mesh file match those defined in the .control file. 
  **_Missing boundary condition type for boundary infolw
  Error in file BoundaryConditions.f90, in line 227_.**
I would greatly appreciate any advice or suggestions. Thank you!

Best,
Dewu

**Comment by Dewu-Yang** [comment]
Sorry, due to my negligence, there was an issue with the boundary conditions I defined. I will retest them again.

**Comment by loganoz** [comment]
Hi @Dewu-Yang, 

Yes, it seems you had a typo in the definition of the "infolw" boundary condition. 
I'll close the discussion. Feel free to open a new one if you have any problem.

Best,
Gonzalo
